src/internal/operators/endWith.ts

```diff
@@ -62,5 +63,5 @@ export function endWith<T, Z = T>(...array: Array<Z | SchedulerLike>): OperatorF
  * @owner Observable
  */
 export function endWith<T>(...array: Array<T | SchedulerLike>): MonoTypeOperatorFunction<T> {
-  return (source: Observable<T>) => concat(source, ...(array as any[])) as Observable<T>;
+  return (source: Observable<T>) => concat(source, of(...(array as any[]))) as Observable<T>;
```

Comment:
Is the `as any[]` completely unavoidable? Without the cast to `any` this would have been caught at compile time

Reply:
The assertion is not necessary. I've removed it. Thanks.